### ISSUE TITLE
Add pause/play support for media keys (while in another window)

### ISSUE THREAD

**Issue by brianmlatimer** [Stremio/stremio-features#341]
**Is your feature request related to a problem? Please describe.**
I've been using Stremio for years, but a big annoyance is not being able to play/pause video while either not at the computer, or not in the same windows as the stremio player (like playing a video game while stremio plays in the other monitor)

I like multitasking and having to stop what I am doing to click back to stremio and pause is very annoying.

**Describe the solution you'd like**
Add support for media keys to play/pause the video while its not in focus. Most video players do this, like Youtube, Netflix, HBO, etc. Also adding support for the pause button on headphones would be nice.

**Comment by CharlieMaskell** [comment]
I would also love this. I often use Stremio whilst playing games and having to tab out whilst playing a game to pause is frustrating.  Every other major video player supports pause/play hardware keys whilst the player is out of focus.

**Comment by Hazem-Gamall** [comment]
Hopefully they add this feature to the Linux client too.

**Comment by MonzterDev** [comment]
This would be such a nice quality of life feature :D

**Comment by Hazem-Gamall** [comment]
I have opened a pull request implementing that functionality on Linux for gnome-based Desktop environments, feel free to check it out [here](https://github.com/Stremio/stremio-shell/pull/350).

**Comment by TheJeran** [comment]
Bumping

**Comment by DaniBedz** [comment]
This would be great. I'm only using the web version due to hotkey support - it'd be nice to use the osx client instead.

**Comment by taaktuuq** [comment]
This would be an amazing addition.

**Comment by freakpants** [comment]
Yes, this would be great on the windows client. Very used to just tapping my bluetooth headset in plex while getting something in the kitchen etc.

**Comment by vUbev** [comment]
i cant believe a basic function like this is still missing. I wish i was smart enough to contribute and make it happen lmao

**Comment by virtualgurel** [comment]
+1

**Comment by mischief957** [comment]
Please god add this. It's a big deal for me cause I need to be able to pause often with my device out of reach.

**Comment by gulltronica** [comment]
Also requesting this... It's the only app I use that lacks this feature.

**Comment by kurutheguru** [comment]
pls add this already 👍

**Comment by GayMerGirlGwen** [comment]
I also agree that this basic media function be given some priority, it is rather illogical for this to remain an issue

**Comment by smashedjuicebox** [comment]
Please

**Comment by simoxon** [comment]
I would like to second this. Stremio is pretty much perfect as it is from what I can see, but not being able to pause the video while in another window is a pain.

**Comment by sbaghdadi** [comment]
Also a plus one from me, please add this feature

**Comment by Krullexneo** [comment]
Please add this <3 I love the Stremio player but not being able to easily Play/Pause with media keys is awful :'(

**Comment by alexballistic** [comment]
still wishing this is added......
